_posts/2025-05-16-Introduction_to_bidirectional-languages-in-your-website.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: Introduction to bidirectional languages in your website
+author: Kami Nasri
+excerpt: >
+  What is RTL? LTR? What are Bidirectional languages? Let's take a look and see how to make your website support these languages.
+tags:
+  - RTL
+  - Frontend
+  - Localization
+---
+
+# Introduction to bidirectional languages in your website
+
+## What are bidirectional languages?
+
+Bidirectional languages or BiDi languages for short refers to Bidirectional languages which contains both left-to-right (LTR) and right-to-left (RTL) writing orders.
+
+A website supporting BiDi text is just not all about translation or mirroring text depending on the language used. BiDi support is about expected behavior. On addition to displaying text there should be also ability to input text in either RTL or LTR depending on the language used and in some cases the ability to display different calendars, e.g. [Solar Hijri Calendar](https://en.wikipedia.org/wiki/Solar_Hijri_calendar)
```

Comment:
is just not all -> is not just

Better word ordering and all is unnecessary in the sentence in my opinion

Reply:
On addition -> In addition

Reply:
be also -> also be

Reply:
. missing at the end of the sentence.